modules/acquia_cms_search/acquia_cms_search.module

```diff
@@ -161,3 +162,80 @@ function acquia_cms_search_search_api_server_update(EntityInterface $entity) {
 function acquia_cms_search_form_acquia_cms_solr_search_form_alter(array &$form) {
   $form['#submit'][] = AcquiaSearchFacade::class . '::submitSettingsForm';
 }
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_search_modules_installed(array $modules) {
+  $module_handler = Drupal::moduleHandler();
```

Comment:
@panshulK I think we do not need to use module_handler service as `$array` contains the modules which were installed.
We can use that. Ex:
```suggestion
$acmsModules = ["acquia_cms_person", "acquia_cms_place", "acquia_cms_article", "acquia_cms_event", "acquia_cms_page"];
if (array_intersect($acmsModules, $modules)) {
  // Create facets.
}
```

Reply:
Done